action.yml

```
@@ -46,9 +46,11 @@ runs:
     - name: Check if version string contains 'v' and strip it out
       shell: bash
       run: |
-        echo "VERSION=${{ inputs.version }}" >> $GITHUB_ENV;
+        VERSION=${{ inputs.version }}
         if [ "${VERSION#*v}" != "$VERSION" ]; then
           echo "VERSION=${VERSION#*v}" >> $GITHUB_ENV;
+        else
+          echo "VERSION=${VERSION}" >> $GITHUB_ENV;
```

Doesn't something like this work?

```suggestion
        VERSION=${{ inputs.version }}
        echo "VERSION=${VERSION#*v}" >> $GITHUB_ENV
```

i.e. always remove the leading `*v` (and it won't remove it if not matched). At least it seems to work in shell script :thinking:

---

Huh so it does :thinking: Somehow I was under the impression that by telling it to take everything after v, it would just return an empty string if no v was present